src/Components/GroupPage/GroupPage.js

```diff
@@ -55,9 +58,11 @@ const GroupPage = () => {
             placeholder="Search groups..."
             onChange={handleGroupSearch}
           />
-        {groups.map((group) => (
-          <div key={group.id}>{group.groupName}</div>
-        ))}
+        {
+        groups.groups.map((group) => (
+          <div className='short-tile' key={group.id}>{group.name}</div>
+        ))
+      }
       </div>
 
       <div className="half-card">
```

Comment:
I might be totally crazy this morning, but I don't see a return anywhere in this component...?